movePlayerScript.js

```diff
@@ -0,0 +1,14 @@
+var storedKeyPress = 'a';
+
+function moveLeg (keyPress, legChoice) {
+    $(document).keydown(function(e){
+        if(e.keyCode == keyPress && e.keyCode !== storedKeyPress) {
+            $(legChoice).animate({left: "+=12px"}, 0.01, function(){
```

Comment:
Can the amount of movement be stored in a variable which can be changed dynamically? Might make future development more dynamic.

Reply:
why? the legs need to move the same distance as its alternate, when implemented within the game the coder can choose the distance they want the leg to move if 12px is too much not enough just as they have to choose the html element they use it on

Reply:
If we pull out this variable then we might ba able to change it dynamically, like when our asian freind picks up an opium pipe. This value could go up by say 5 px for 5 seconds then. 
NOt strictly MVP.. but still, it would be a minor change for a lot of added dynamism.